### PR TITLE
NOTICK Set Hikari logging to WARN

### PR DESCRIPTION
**Pull request by lankydan**

### CODE REVIEW

**Review by LWogan — DISMISSED** [review]
LGTM

File: charts/corda/log4j2.xml [hunk 1]
```diff
@@ -17,9 +17,9 @@
         </logger>
 
         <!-- log warn only for these 3rd party libs -->
-        <Logger name="org.apache.aries.spifly" level="warn" />
```
Comment (davidcurrie) [inline charts/corda/log4j2.xml:20]: we seem to have lost spifly...

Comment (lankydan) [reply charts/corda/log4j2.xml:20]: 🤦

**Review by davidcurrie — APPROVED** [review]
👍

**Pull request merged**